charts/kube-state-metrics/Chart.yaml

```diff
@@ -7,7 +7,7 @@ keywords:
 - prometheus
 - kubernetes
 type: application
-version: 4.18.0
+version: 4.18.1
```

Comment:
As this is a new feature, can you bump chart's version to 4.19.0 please

Reply:
@dotdc done